cryptostore/data/gd.py

```diff
@@ -0,0 +1,158 @@
+'''
+Copyright (C) 2018-2020  Pierre Juillard - pierre.juillard@gmail.com
```

Comment:
I'm not sure of the implications of merging in code someone else claims a copyright to. Would it prevent me from modifying it? Distributing it? etc. Its definitely fair to leave your name here (Original implementation by .... etc) and its also fine to not contribute this (you can close the PR). I'm just saying, I wont take any code anyone else claims a copyright to.